wiki/Tournaments/MWC/2021_4K/en.md

```diff
@@ -59,71 +59,81 @@ The osu!mania 4K World Cup 2021 is run by the [osu! team](/wiki/People/The_Team)
-    - Disqualification from the current and future official tournaments until appealed.
-22. Any modification of these rules will be announced.
+    - Disqualification from the current and future official tournaments, until appealed.
+21. The Tournament Management reserves the right to modify these rules at any moment. Any such changes will be announced in advance.
 
 ### Tournament registration
 
 1. Every user interested in joining their country's team signs up individually.
    - Tournament Management will create a list of potential candidates for a country's team.
-   - Tournament Management declares one candidate to the captain of the country's team, albeit temporarily.
-   - The declared captain can form their team from the candidate list of their country.
-2. To ensure valid and serious registrations, every registered user will be checked by the Tournament Management.
+   - Tournament Management will declare one candidate captain of their country's team.
+2. The declared captain will then form their team from the candidate list of their country.
+   - Captains are expected to choose team members with honesty and good will, with the aim of fielding the strongest team possible.
+   - Captains are allowed to exclude themselves from the team list.
+   - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms.
+   - Captains may choose, with the approval of the Tournament Management, to delegate their role to other prospective team members.
+3. To ensure valid and serious registrations, every registered user will be manually checked by the Tournament Management.
    - Every registered user will be assigned to their respective country's candidate list.
-   - To be successfully accepted on the list, you have to ensure that your global osu!mania 4K performance ranking is above 5000.
-   - To be successfully accepted on the list, you have to ensure that you did not violate the [osu! community rules](/wiki/Rules) within the last 12 months.
-3. All successfully formed teams will be published after the Registration Phase.
-4. Only the 32 potentially strongest countries will participate. The potential strength of a country is determined by the online statistics of all valid candidates.
-5. Mapset selectors must not participate as a player in this tournament.
+   - To be successfully accepted on the list, players are required to be placed #5000 and up on the global osu!mania 4K performance ranking, and have not violated the [osu! community rules](/wiki/Rules) within the last 12 months.
+4. All successfully formed teams will be published after the Registration Phase.
+   - A team should have at least 3 players registered to have the chance to play the Qualifier stage.
+   - The top 32 seeded teams will qualify to the Round of 32.
+5. Organisers, Mappool Selectors and referees must not participate as a player in the tournament.
 
-### Stage instructions
+### Qualifier instructions
```

Comment:
As mentioned in the previous comment, please confirm if map style distributions will be a thing or not.

Reply:
If it takes time to confirm though let me know so we can update the rest at least.

Reply:
It's going to take a while to confirm it and there's some info that needs to be updated ASAP. I think it's fine if we leave this for later (probably when the pool is released).